cfgov/v1/migrations/0004_eventarchivepage.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import models, migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('v1', '0003_add_tags-authors_for_cfgovpage'),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='EventArchivePage',
+            fields=[
+                ('cfgovpage_ptr', models.OneToOneField(parent_link=True, auto_created=True, primary_key=True, serialize=False, to='v1.CFGOVPage')),
```

Comment:
Not sure. This is an autogenerated migration script. It might be a good idea for the future when the database become permanent.